### DISCUSSION TITLE
Function to disable the extension from the website side

### DISCUSSION THREAD

**Discussion by DygDyg** [darkreader/darkreader#13144] [Q&A]
Hi, I have my own website that uses a dark theme. Is there something I can add to the page header so that your extension recognizes this and doesn't change the colors, as it breaks the page background? It’s quite tedious to explain to all the users to disable the extension for everything to work.

Привет, у меня есть свой сайт, он использует тёмную тему, можно ли что то добавить в заголовок страницы, чтобы ваше расширение понимало это и не перекрашивало его по умолчанию, а то оно ломает фон страницы. А то запарно объяснять всем пользователям сайта, что нужно отключить расширение чтобы всё работало.

**Comment by Sclafus** [comment]
Hey @DygDyg, have you already looked at this section? https://github.com/darkreader/darkreader/blob/main/CONTRIBUTING.md#disabling-dark-reader-on-your-site
